website/en/docs/frameworks/redux.md

```diff
@@ -124,6 +124,44 @@ function bar(value: boolean): BarAction {
 }
 ```
 
+##### Typing Redux thunk actions <a class="toc" id="toc-typing-redux-thunk-actions" href="#toc-typing-redux-thunk-actions"></a>
+
+In order to type your Redux [thunk actions](http://redux.js.org/docs/advanced/AsyncActions.html#async-action-creators),
+you'll add types for `ThunkAction` as a function `Dispatch`, and `GetState`. `GetState` is a function that returns an `Object`. `Dispatch` accepts a disjoint union of `Action`, `ThunkAction`, `PromiseAction` and `Array<Action>` and can return `any`.
```

Comment:
`PromiseAction` and `Array<Action>` require additional middleware to be installed.